functions.php

```diff
@@ -24,5 +25,5 @@ function getDataFromDB(PDO $db)
 {
     $query = $db->query("SELECT * FROM `computerMice` WHERE (`name` IS NOT NULL AND `brand` IS NOT NULL);");
```

Comment:
`*` is bad practise, you should list out the field names. using `*` is really slow